### PR TITLE
The green color used for the headings on the [Collaborate] page changed to White…

### LINKED ISSUE #5266: The green color used for the headings on the [Collaborate] page needs to be changed to White.
### Description
The green color used for the headings on the [Collaborate] page needs to be changed to White.
https://layer5.io/cloud-native-management/meshmap/collaborate

### Expected Behavior
The green color used for the headings on the [Collaborate] page needs to be changed to White.

### Environment:
- Host OS:
- Browser:

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Ashparshp**
**Description**
The green color used for the headings on the [Collaborate] page changed to White.
#5266
https://layer5.io/cloud-native-management/meshmap/collaborate
This PR fixes #5266

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 6355522430b9de1f2ca6c507222088c22a7addf4 at: https://659732c4ebd096465fb143f3--layer5.netlify.app

**Comment by vishalvivekm** [comment]
@Ashparshp 
Today, we've our weekly Websites meeting at 6:30 PM IST : https://meet.layer5.io/websites.
Let's get feedback on your work in today's call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

File: src/sections/Meshmap/Meshmap-collaborate/collaboration-feature-team.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ const CollaborationFeatureWrapper = styled.div`
 
     h2 {
       span {
-        color: #00b39f;
+        color: #FFFFFF;
```
Comment (ritiksaxena124) [inline src/sections/Meshmap/Meshmap-collaborate/collaboration-feature-team.js:51]: Use theme referencing to change the color

File: src/sections/Meshmap/Meshmap-collaborate/collaboration-feature-work.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ const CollaborationFeatureWrapper = styled.div`
 
     h2 {
       span {
-        color: #00b39f;
+        color: #FFFFFF;
```
Comment (ritiksaxena124) [inline src/sections/Meshmap/Meshmap-collaborate/collaboration-feature-work.js:42]: Use theme referencing to reference the color

Comment (Ashparshp) [reply src/sections/Meshmap/Meshmap-collaborate/collaboration-feature-work.js:42]: I've chosen colors from the Layer5 design itself.

**Comment by l5io** [comment]
🚀 Preview for commit d650dd0d975aadccdff1136057a859f1ed7f1bdb at: https://65b1558ecbb0c02e4c55ffde--layer5.netlify.app

**Comment by ritiksaxena124** [comment]
Thank you for this @Ashparshp 🙌

**Pull request merged**